<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeAvatar</class>
 <widget class="QDialog" name="ChangeAvatar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>376</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>376</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>376</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #37474f;
    background-image: url(:/textures/bottom_bg.png);
    background-repeat: repeat-x;
    background-position: bottom;
}

.QStackedWidget &gt; QWidget {
    border: none;
background-color: transparent;
outline: 0px;
}


#bottom_widget {
    min-height: 67px;
max-height: 67px;

}

QWidget {
	font: 11px &quot;Open Sans&quot;;
}

QLineEdit, QComboBox {
    padding-left: 3px;
    min-width: 145px;
    max-width: 145px;
    min-height: 24px;
    max-height: 24px;
    border: 1px solid transparent;
    border-radius: 2px;
}

QPlainTextEdit{
    min-width: 148px;
    max-width: 148px;
    
    border-radius: 2px;
}
#input_day, #input_month, #input_year {
    min-width: 36px;
}

QPushButton {
    font: 14px &quot;Open Sans&quot;;
    color: rgba(250, 250, 250, 250);
}
QPlainTextEdit {
    border: 1px solid;

}

QComboBox {
    color: #7f7f7f;
}
QComboBox:on { /* shift the text when the popup opens */
    color: #191919;
}

QComboBox::down-arrow {
	
    max-width: 20px;
    max-height: 20px;
    min-width: 20px;
    min-height: 20px;
}
QComboBox:open::drop-down {
	background-position: center;
}
QComboBox:open {

}

QComboBox::drop-down {
    
	background-image: url(:/textures/MainPage/bttn3x1_scroll_down.png);
    max-width: 20px;
    max-height: 20px;
    min-width: 20px;
    min-height: 20px;
    border: none;
}

QLabel {
    color: rgba(250, 250, 250, 250);
    font: 550 11px &quot;Open Sans&quot;;
}

QLineEdit:focus {
    border: 1px solid #00aeef;
}

QToolTip {
    border: 1px solid #00aeef;
    background-color: #ffffff;
font: 11px &quot;Open Sans&quot;;
color: black;

}
</string>
  </property>
  <widget class="QStackedWidget" name="stackOfContent">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>309</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>309</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>309</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="ns_page::SelectAvatar" name="page_avatarSelect">
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>309</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>309</height>
     </size>
    </property>
   </widget>
   <widget class="ns_page::UploadAvatar" name="page_avatarUpload"/>
  </widget>
  <widget class="QWidget" name="bottom_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>640</width>
     <height>67</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>67</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1231231</width>
     <height>67</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>17</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0" alignment="Qt::AlignVCenter">
     <widget class="QPushButton" name="bttn_reject">
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>34</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>124</width>
        <height>34</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 43, 49, 255), stop:0.93 rgba(69, 89, 99, 255), stop:0.95 rgba(131, 171, 190, 255), stop:1 rgba(131, 171, 190, 255));
border: 1px solid black;
border-radius: 3px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(205, 55, 33, 215), stop:0.33 rgba(69, 24, 22, 225), stop:0.94 rgba(69, 89, 99, 255), stop:0.95 rgba(121, 131, 150, 255), stop:1 rgba(131, 171, 190, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(21, 13, 14, 255), stop:0.93 rgba(3, 4, 2, 255), stop:0.95 rgba(33, 36, 41, 255), stop:1 rgba(41, 41, 44, 255));
}</string>
      </property>
      <property name="text">
       <string>@bttn_cancel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QStackedWidget" name="stackOfButtons">
      <property name="minimumSize">
       <size>
        <width>174</width>
        <height>34</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>174</width>
        <height>34</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <widget class="QPushButton" name="bttn_ok">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>124</width>
          <height>34</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>124</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>124</width>
          <height>34</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
border: 1px solid #191919;
border-radius: 3px;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 121, 40, 255), stop:0.94 rgba(0, 199, 83, 255), stop:0.95 rgba(0, 250, 158, 255), stop:1 rgba(0, 250, 158, 255));
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 121, 40, 255), stop:0.95 rgba(0, 250, 158, 255), stop:1 rgba(0, 250, 158, 255));
}

QPushButton:pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 41, 20, 255), stop:0.95 rgba(0, 90, 55, 255), stop:1 rgba(0, 90, 50, 255));
}
QPushButton:disabled {
    border: 1px solid rgba(25, 25, 25, 166);
    color: rgba(250, 250, 250, 166);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 121, 40, 166), stop:0.94 rgba(0, 199, 83, 166), stop:0.95 rgba(0, 250, 158, 166), stop:1 rgba(0, 250, 158, 166));
}


</string>
        </property>
        <property name="text">
         <string>@bttn_OK</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="bttn_save">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>124</width>
          <height>34</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>124</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>124</width>
          <height>34</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
border: 1px solid #191919;
border-radius: 3px;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 121, 40, 255), stop:0.94 rgba(0, 199, 83, 255), stop:0.95 rgba(0, 250, 158, 255), stop:1 rgba(0, 250, 158, 255));
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 121, 40, 255), stop:0.95 rgba(0, 250, 158, 255), stop:1 rgba(0, 250, 158, 255));
}


QPushButton:pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 41, 20, 255), stop:0.95 rgba(0, 90, 55, 255), stop:1 rgba(0, 90, 50, 255));
}

</string>
        </property>
        <property name="text">
         <string>@bttn_save</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ns_page::SelectAvatar</class>
   <extends>QWidget</extends>
   <header location="global">pages/page_SelectAvatar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ns_page::UploadAvatar</class>
   <extends>QWidget</extends>
   <header location="global">pages/page_UploadAvatar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stackOfContent</sender>
   <signal>currentChanged(int)</signal>
   <receiver>stackOfButtons</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttn_reject</sender>
   <signal>clicked()</signal>
   <receiver>ChangeAvatar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>622</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttn_ok</sender>
   <signal>clicked()</signal>
   <receiver>ChangeAvatar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttn_save</sender>
   <signal>clicked()</signal>
   <receiver>ChangeAvatar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
